tests/registration/Registration.Service.Tests/BusinessLogic/RegistrationBusinessLogicTest.cs

```
@@ -2918,6 +2918,23 @@ public async Task GetDocumentAsync_WithWrongUser_ThrowsForbiddenException()
         ex.Message.Should().Be($"The user is not permitted to access document {documentId}.");
     }
 
+    [Fact]
+    public async Task GetDocumentAsync_WithConfirmedApplicationStatus_ThrowsForbiddenException()
```

The title says if status confirmed it should throw an error but currently the test setup is that confirmed is false, please adjust the setup and the business logic afterwards.

---

Adjusted and setup changed